app/components/avo/fields/common/files/view_type/grid_item_component.html.erb

```diff
@@ -1,27 +1,23 @@
 <div id="<%= dom_id file %>" class="relative min-h-full max-w-full flex-1 flex flex-col justify-between space-y-2">
-  <% if file.present? %>
```

Comment:
The field wrapper will intercept and not render this component. That's why we don't need the `file.present?` line.